portfolio/src/main/webapp/index.html

```diff
@@ -69,6 +69,10 @@ <h1>Michaela Murray's Portfolio</h1>
       <br/><br/>
       <input type="submit"/>
     </form>
+    <form method="POST" action="/delete-data">
+      <input type="number" name="delete-num" />
```

Comment:
Wouldn't be a bad idea to have a label showing the user this textbox contains the number of comments they would like to delete.

Reply:
thanks! updated!

Reply:
Can you associate this [label with the input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)? Either explicitly or implicitly. This will help accessibility.

Reply:
I'm not sure why a label is not present yet, but I did add a label, which is hopefully reflected by the below commits ;)

Reply:
did this get done? looks like the label still doesn't have a `for` attribute...

Reply:
should be addressed now!